src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -91,21 +91,28 @@ function NameAndPicture({ navigation, route }: Props) {
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
   const showGuidedOnboarding = useSelector(showGuidedOnboardingSelector)
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
+  const showNameGeneratorButton = nameType === OnboardingNameType.AutoGen
 
   useLayoutEffect(() => {
     navigation.setOptions({
       headerTitle: () => {
         let pageTitleTranslationKey
-        if (showGuidedOnboarding) {
+        pageTitleTranslationKey = choseToRestoreAccount
```

Comment:
if `showGuidedOnboarding` is true and `choseToRestoreAccount` is also true, the new code chooses a different page title. Is that intended? Seems to change the behavior of a different experiment..

Reply:
It is intended; it is indeed a place of confusion;
The trusted guide covers title for both restore and new, and this implementation is consistent with that of the experiment already started

Reply:
i misread the boolean logic